.github/composite_actions/get_platform_parameters/action.yml

```diff
@@ -5,62 +5,99 @@ inputs:
     required: true
     type: string
   xcode_version:
-    description: 'The version of Xcode. Valid values are 14.3 and 15.0'
-    required: true
+    description: "The version of Xcode. Available aliases are 'latest' and 'minimum'"
+    default: 'latest'
+    type: string
+  destination:
```

Comment:
Is this new `destination` input meant to override the one that is automatically determined by the Xcode version?

When is it used or needed?

Reply:
Yes, it is used to define the exact destiation. In case of `latest` or `minimum`, you can use the default empty string value and the script will try to use the predefined destination mappings.
This input option becomes essential when testing a particular Xcode version that is neither `latest` nor `minimum`. For example, when testing a beta Xcode version and specifying the beta SDK version in the `destination` input.